pkg/config/config.go

```diff
@@ -378,3 +378,18 @@ func (c *ConfigInfo) DeleteApplication(application string, project string) error
 	}
 	return nil
 }
+
+// DeleteProject deletes applications belonging to the given project from the config file
+func (c *ConfigInfo) DeleteProject(projectName string) error {
+	for i := len(c.ActiveApplications) - 1; i >= 0; i-- {
+		// remove current item from array
```

Comment:
shouldn't this comment be before `c.ActiveApplications = append(c.Act ....` line?

Reply:
fixed